xla/service/gpu/fusions/transpose_test.cc

```diff
@@ -243,6 +243,101 @@ TEST_F(TransposeTest, ThreadIndexingPartialBlock) {
       )"));
 }
 
+TEST_F(TransposeTest, SameInputIndexingForRealHeroAndSideOutput) {
```

Comment:
It seems this test needs some adjustments in the test expectations after latest merge. In our imported version of this PR, the test is failing